test-fixtures/src/main/resources/fixtures/multi-column-select-in-where/Test.s

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE foo(
+  id INTEGER NOT NULL,
+  name TEXT NOT NULL
+);
+
+SELECT *
+FROM foo
+WHERE (id, name) IN ?;
+
+SELECT *
+FROM foo
+WHERE (id, name) IN (?, ?);
```

Comment:
Should it support `(?, ?)` or `((?, ?), (?, ?))` instead?

Reply:
I would like `?` to not specify the exact length in sql but during runtime.

Reply:
Oh, already supported. Okay, I would not support `(?, ?)` or any other syntax at all. I think, this "sql code" would be annoying to write but just `?`.

Reply:
Alright removed `LP expr ( COMMA expr ) * RP`